utils.py

```diff
@@ -420,6 +437,13 @@ def get_human_readable_time_string(time_msec):
 def are_datetimes_close(later_datetime, earlier_datetime):
     """Given two datetimes, determines whether they are separated by less than
     feconf.PROXIMAL_TIMEDELTA_SECS seconds.
+    
+    Arg:
+        later_datetime: function. Later date and time.
+        earlier_datetime: function. Earlier date and time.
+    
+    Return:
+        float. Whether they are separated by less than feconf.PROXIMAL_TIMEDELTA_SECS seconds.
```

Comment:
I think the return type is incorrect, can you please check?

(Reverify all args and return types and make sure they are correct, you can use `type` and `print` statement to check the correct types or read the documentation/code of the method which uses these util methods. To find the place where these methods sre used you can try `git grep <method-name>`)

Reply:
@DubeySandeep Bhaiya actually i'm using git Kraken(no command line) for edit as i'm a beginner , so bhaiya via command i couldn't check all that.
Actually bhaiya i don't think i'm doing all it right way because i haven't check the code in any compiler. If you are not busy can you guide me what to do or an alternate from which i can learn.

Reply:
done